site/gatsby-site/src/components/taxa/Taxonomy.js

```diff
@@ -57,8 +57,20 @@ const Taxonomy = ({ taxonomy, incidentId, canEdit }) => {
     setShowBanner(true);
   };
 
+  useEffect(() => {
```

Comment:
Leftover console.log

Reply:
I'm currently getting a test failure that only occurs on Netlify, so I've pushed log statements to try to diagnose the problem. This will all be reverted soon.

Reply:
If you have any ideas, the failure is on:

```
Should show the taxonomy form of resources:
1:41:50 PM:      AssertionError: Timed out retrying after 30000ms: Expected to find element: `[data-cy="resources"]`, but never found it.
```

Reply:
This branch has schema changes, so the tests will fail. However, I set up another Netlify project that uses my dev database on which the tests should pass. That deploy log is [here](https://app.netlify.com/sites/lmcnulty-staging/deploys/62d1a4e07f85b10009bb8a65).

Reply:
Those errors above are independent of the `schema.json`. It is the `gatsby-source-mongodb` plugin that can't infer the data type of those fields. The same issue you had the other day.

Reply:
Regarding the test you mention, that one is flaky af, maybe setting up Cypress Dashboard could help you. It will record videos of what it is actually happening.

Reply:
This PR also contains the migrations that add the field. The plugin can't infer their type because they don't exist on the production db.

Reply:
Yes, and the error mentions that you can add schema customization: 

https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/gatsby-node.js#L164

I had to do this a few times, you'll notice.